shared/js/content-scripts/click-to-load.js

```diff
@@ -555,11 +599,33 @@
                     }
 
                     fbContainer.appendChild(fadeIn)
-                    // default case is this.clickAction.type === 'originalElement'
-                    let fbElement = originalElement
 
-                    if (this.clickAction.type === 'iFrame') {
+                    let fbElement
```

Comment:
Yea, there's quite a few comments/variable names like that. I'd rather improve those as a follow-up since this is already a pretty big patch, WDYT? (I have filed a task for that in Asana already.)